spec/api-subframe-spec.js

```diff
@@ -74,15 +76,38 @@ describe('renderer nodeIntegrationInSubFrames', () => {
       it('should correctly reply to the nested sub-frames with using event.reply', async () => {
         const detailsPromise = emittedNTimes(remote.ipcMain, 'preload-ran', 3)
         w.loadFile(path.resolve(__dirname, 'fixtures/sub-frames/frame-with-frame-container.html'))
-        const [,, event3] = await detailsPromise
+        const [, , event3] = await detailsPromise
         const pongPromise = emittedOnce(remote.ipcMain, 'preload-pong')
         event3[0].reply('preload-ping')
         const details = await pongPromise
         expect(details[1]).to.equal(event3[0].frameId)
       })
+
+      if (webPreferences.contextIsolation) {
```

Comment:
Because it's bitten us in the past, can we invert the test when `contextIsolation` is `false`?

E.g. 

```js
if (webPreferences.contextIsolation) {
  expect(result).to.be.null();
} else {
  expect(result).to.equal(true);
}
```